Easydict/Feature/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -527,6 +527,7 @@ - (void)focusInputTextView {
         [NSApp activateIgnoringOtherApps:YES];
         
         [self.baseQueryWindow makeFirstResponder:self.queryView.textView];
+        self.queryView.textView.selectedRange = NSMakeRange(0, self.queryText.length);
```

Comment:
What is this code used for?

Reply:
This is for automatically selected the previous text. So if users want to query a new word, they can just start typing. I dont know if this is a expected behavior in Input Translate when clear is NO in settings.

Reply:
I see, a user request this before https://github.com/tisfeng/Easydict/issues/300 , but at the time I wasn't sure if it was a real feature so I ignored it.

Now with the new settings, it's easy to add a settings option, so if users really need this feature, we can consider adding an option.

Reply:
do you mean to add a new option to control the auto-select feature? or is there already a suitable one?

Reply:
Add a new option.

Reply:
Done